### ISSUE TITLE
[A11y Design Review]: MHV Vaccines & Health history landing page

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#54928]
## Auditor(s)
Sara Smith

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [x] WCAG 2.1 AA
  - [x] WCAG 2.1 AAA
  - [x] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [x] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav) (Not applicable)
  - [ ] Annotations for Tab order, reflow and reading order (if needed)
- [ ] Accessible Name Annotations (Not applicable) 
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [x] Language (ease of reading)
- [x] HTML Tags
  - [x] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [x] [Heading order](https://accessibility.18f.gov/headings/)
   - [x] Annotations (if needed)
  - [x] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [x] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

### Supporting artifacts
- Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
- Link to or attach user flows:
     - [User flow](https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/a/MybMyWw) This is tentative, pending IA research findings. 
- Link to prototype or mockups (these will be adjusted continuously until usability testing; more screens may be added later):
     - Health history (Medical records) landing page [Desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/MybDm39)
     - Vaccines list page [Desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/8B956FE2-A63F-48C8-B8F1-FF93329ACE0E)
     - Selected vaccine details page [Desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/D898C623-9070-42F2-841A-A859BFC98D09)
     - [Modal: Choosing the file type to download
](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/D3F34518-813E-48BD-8367-48868DB4EB4E)
     - [Download success/fail alerts](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E9BD7A74-E456-4DB3-AD2F-18CF8814E73F)
## Prototype Link

[Link to prototype](insert link here)

### Concise issue description

![Image 4-14-23 at 2.07 PM.jpg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/ac2973a4-d73b-4ad1-84b4-cfaa7b190000)

"Print or download list" button is not a design system component. Because this is new/experimental this would need to go through DSC (Design System Council) 

Questions from @sara-amanda on Modal preview

![Image 4-14-23 at 11.48 AM.jpg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/ce4fed69-6b1e-480f-92fc-d6a85bb7bbd3)
First Modal "Download":
"Martha mentioned that there have been recent zoom studies that show these modals just may not be seen at all."

798-6e89-499f-b87b-93ae3234c57f) Second Modal:
"I realize why we need the second modal, due to public computer usage; however, that creates two modals that have the potential to not be seen and have focus issues.
Will that only show on public computers?
If so, there is less chance of a zoom magnification or screen reader on a public machine, and could be ok.
It should still withstand super large font settings"

[Link to Modal discussion via Slack](https://dsva.slack.com/archives/C03Q2UQL1AS/p1680625442522479)

## Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by laurwill** [comment]
Hi @sara-amanda , 
I wanted to get an accessibility ticket set up for the vaccines domain, because I had a couple Qs on one of the sketch screens. 

https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/7y00pyY

That screen references the best file type to choose for assistive devices — I believe the language came from an existing tool, but wanted to double check that the way we're describing the options sounds right!

**Comment by sara-amanda** [comment]
If you’re printing, choose PDF (.pdf)
If you’re using assistive technology such as a screen reader, screen enlarger, or braille display, choose a Text file (.txt)

@laurwill

**Comment by sara-amanda** [comment]
## CAIA New Related Ticket

### Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by SarahKay8** [comment]
@laurwill Hi Laura! Design review complete. Please close out once reviewed.

**Comment by SarahKay8** [comment]
Hi @alexiawunder. Design review complete. Can you close this out once reviewed. No further feedback to add.
